src/tools/compiletest/src/common.rs

```diff
@@ -613,7 +614,7 @@ pub struct Config {
     ///
     /// FIXME: perform stronger validation for this. There are editions that *definitely* exists,
     /// but there might also be "future" edition.
```

Comment:
```suggestion
```

While we didn't make all illegal states unrepresentable (e.g., `Edition::Year(1234)`), I think we can remove the FIXME unless you plan on switching away from a `usize` to an explicit enum impl'ing `Ord` eventually.